workers/unity/Packages/com.improbable.gdk.tools/RedirectedProcess.cs

```diff
@@ -17,31 +17,114 @@ public class RedirectedProcessResult
         public List<string> Stderr;
     }
 
+    [Flags]
+    public enum ProcessOutputOptions
+    {
+        /// <summary>
+        ///   <para>No redirected output, only custom outputProcessors are used</para>
+        /// </summary>
+        None = 0,
```

Comment:
`NoRedirect` ?

Reply:
This is set of flags, not direct enum, so `.NoRedirect | .RedirectStdOut` is a valid assignment, though doesn't make a lot of sense semantically. None also doesn't look great in this case, but I am not sure what would be a good naming

Reply:
Alternatively you could rename the enum to `RedirectBehaviour` in which case `None` makes more sense 💯